ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/util/DataStructureUtil.java

```diff
@@ -511,6 +513,16 @@ public ExecutionPayload emptyExecutionPayload() {
             List.of());
   }
 
+  public Transaction randomExecutionPayloadTransaction() {
+    final TransactionSchema schema =
+        new TransactionSchema(
+            spec.forMilestone(SpecMilestone.BELLATRIX)
+                .getConfig()
+                .toVersionBellatrix()
+                .orElseThrow());
```

Comment:
Probably better to get this from the spec schema definitions:
```suggestion
    final TransactionSchema schema =
        SchemaDefinitionsBellatrix.required(spec.getGenesisSchemaDefinitions())
            .getExecutionPayloadSchema()
            .getTransactionSchema();
```

You'll need to make that `getTransactionSchema()` method public as well but I think that's a good change since the `ExecutionPayloadSchema` is able to guarantee the schema type for transactions and we lose some detail in the generics when getting the element schema from the transactions list.

We don't tend to use the transaction schema directly in production, hence it gets a bit buried here.